fotogalleri/gallery/templates/modals/create_path.html

```diff
@@ -7,20 +7,20 @@
 {% endblock %}
 
 {% block modalcontent %}
+<form id="new-path-form">
   <div class="modal-card">
     <header class="modal-card-head">
       <p class="modal-card-title">Add path</p>
       <button id="new-path-form-close" class="delete" aria-label="close"></button>
     </header>
     <section class="modal-card-body">
-      <form id="new-path-form" method="POST">
-        <input id="new-path" name="path" placeholder="Insert new path">
-      </form>
+      <input id="new-path" name="path" placeholder="Insert new path">
     </section>
     <footer class="modal-card-foot">
       <div id="new-path-button">
-        {% include 'components/styled_button.html' with button_text="Create new path" url_name="javascript:;" sizes="is-size-6" only %}
+        {% include 'components/styled_button.html' with type="button" button_text="Create new path" url_name="javascript:;" sizes="is-size-6" only %}
```

Comment:
I don't think `type="button"` does anything here, there are no parameters in `components/styled_button.html` named `type` (however, there is `button_type`).

Reply:
Ah, removed!